perception/CMakeLists.txt

```diff
@@ -74,10 +77,12 @@ include_directories(SYSTEM
                     ${Boost_INCLUDE_DIRS}
                     ${GLEW_INCLUDE_DIR}
                     ${GLUT_INCLUDE_DIR}
+                    ${X11_INCLUDE_DIR}
                     )
 
 link_directories(${Boost_LIBRARY_DIRS})
 link_directories(${catkin_LIBRARY_DIRS})
+link_directories(${X11_LIBRARY_DIR})
```

Comment:
The value of `OPENGL_INCLUDE_DIR` is below, so it does not work. (GL/glut.h
is not found).

/System/Library/Frameworks/OpenGL.framework

​




Sent with MailTrack
<https://mailtrack.io/install?source=signature&lang=en&referral=www.kentaro.wada@gmail.com&idSignature=22>

On Sun, Apr 24, 2016 at 7:35 AM, Robert Haschke <notifications@github.com>
wrote:

> In perception/CMakeLists.txt
> <https://github.com/ros-planning/moveit_ros/pull/664#discussion_r60836085>
> :
>
> >                      )
> >
> >  link_directories(${Boost_LIBRARY_DIRS})
> >  link_directories(${catkin_LIBRARY_DIRS})
> > +link_directories(${X11_LIBRARY_DIR})
>
> Better. Could you please try to replace ${X11_INCLUDE_DIR} by
> ${OPENGL_INCLUDE_DIR}?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ros-planning/moveit_ros/pull/664/files/cd0d15f235395578af9813b551c30f465a15b4bf#r60836085>
>



-- 
和田 健太郎 / Kentaro Wada
http://wkentaro.com